package.json

```diff
@@ -4,8 +4,10 @@
     "api"
   ],
   "scripts": {
-    "pretest": "standard .",
+    "lint": "standard . && tsc --noEmit",
     "readme": "node ./bin/generate-docs.js",
+    "tsc": "./bin/generate-typescript-typings.sh",
+    "pretest": "npm run lint",
```

Comment:
I did somewhat similar in https://github.com/socketsupply/socket/pull/368, but also removed `pretest`. I think we don't need this on pretest as well as `standard .` and `gen-doc` (aka `readme`). Anyway, I can rebase after you merge it.